reward/supply_manager.go

```diff
@@ -384,13 +384,14 @@ func (sm *supplyManager) accumulateReward(from, to uint64, fromAcc *database.Acc
 		// Accumulate one block
 		var (
 			header    = sm.chain.GetHeaderByNumber(num)
+			block     = sm.chain.GetBlockByNumber(num)
 			rules     = sm.chain.Config().Rules(new(big.Int).SetUint64(num))
 			pset, err = sm.gov.EffectiveParams(num)
 		)
 		if err != nil {
 			return nil, err
 		}
-		blockTotal, err := GetTotalReward(header, rules, pset)
+		blockTotal, err := GetTotalReward(header, block.Transactions(), nil, rules, pset)
```

Comment:
Is `nil` ok?

Reply:
Missed it.. added at https://github.com/klaytn/klaytn/pull/2187/commits/4e6ca1e64721d5fba43b283000adb82f03078301. thanks!